python/lsst/ctrl/bps/wms/panda/idds_tasks.py

```diff
@@ -150,12 +155,17 @@ def create_raw_jobs_dependency_map(self):
         """
         dependency_map = {}
         for edge in self.bps_workflow.in_edges():
-            dependency_map.setdefault(edge[1], []).append(edge[0])
+            dependency_map.setdefault(self.get_pseudo_input_file_name(edge[1]), []).\
+                append(self.get_pseudo_input_file_name(edge[0]))
+            self.jobs_steps[self.get_pseudo_input_file_name(edge[1])] = \
+                self.bps_workflow.nodes.get(edge[1]).get('job').label

```

Comment:
I didn't quite follow this code, but wondering if this could be self.bps_workflow.get_job(edge[1]).label

Reply:
Thanks for the tip. Fixed.